src/osd/ReplicatedPG.cc

```diff
@@ -5470,8 +5470,8 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	  result = -ENOENT;
 	  break;
 	}
-        uint64_t cookie = op.watch.cookie;
-        entity_name_t entity = ctx->reqid.name;
+	uint64_t cookie = op.watch.cookie;
+	entity_name_t entity = ctx->reqid.name;
```

Comment:
The original code seems not indent correctly. So I try to modify the indent to align with the code below.